diagnostic.md

```diff
@@ -40,25 +60,26 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    to help show what is information is displayed on the page
```

Comment:
Close, but a better answer would be `A serializer controls what data is returned from a request.`. It doesn't necessarily have to be displayed on the page.